sql/src/main/java/io/crate/jobs/AbstractExecutionSubContext.java

```diff
@@ -96,7 +96,6 @@ protected boolean close(@Nullable Throwable t) {
         return false;
     }
 
-    @Override
     final public void close() {
```

Comment:
The FetchContext is closed from outside and I think in that case it's okay. Didn't want to touch the AbstractExecutionSubContext code to split that up.